Keypad.c

```diff
@@ -0,0 +1,223 @@
+/*
+ * Keypad.c
+ *
+ *  Created on: Feb 2, 2020
+ *      Author: Delaney Berger
+ */
+
+#include "Keypad.h"
+#include "delay.h"
+#include "msp.h"
+#include <stdint.h>
+#include "UART.h"
+
+static uint8_t key;
+
+void KeypadInit(void)
+{
+   P5 -> DIR &= ~(C1|C2|C3|C4);                 //Set Columns(input) as interrupts
+   P5 -> IES |= (C1|C2|C3|C4);
+   P5 -> IFG &= ~(C1|C2|C3|C4);
+   P5 -> IE |= (C1|C2|C3|C4);
+   P5 -> SEL1 &= ~(C1|C2|C3|C4);
+   P5 -> SEL0 &= ~(C1|C2|C3|C4);
+   P5 -> REN = (C1|C2|C3|C4);                   // use enable resistor
+   P5 -> OUT &= ~(C1|C2|C3|C4);                 //  pull down resistors
+
+
+   P2 -> DIR = (R0|R1|R2|R3);                   //sets rows as outputs
+/*
+   P2 -> SEL1 &= ~(R0|R1|R2|R3);                //Not sure why but it doesn't work as well with this section of code
+   P2 -> SEL0 &= ~(R0|R1|R2|R3);
+   P2 -> OUT |= (R0|R1|R2|R3);                  // sets it high
```

Comment:
Where do you set Port 2 high? How does the Port 5 interrupt get called without Port 2 ever being high?